src/configuration-wire/configuration-wire-helper.ts

```diff
@@ -0,0 +1,75 @@
+import ApiEndpoints from '../api-endpoints';
+import FetchHttpClient, {
+  IBanditParametersResponse,
+  IHttpClient,
+  IUniversalFlagConfigResponse,
+} from '../http-client';
+
+import { ConfigurationWireV1, IConfigurationWire } from './configuration-wire-types';
+
+export type SdkOptions = {
+  sdkName: string;
+  sdkVersion: string;
+  baseUrl?: string;
+};
+
+/**
+ * Helper class for fetching and converting configuration from the Eppo API(s).
+ */
+export class ConfigurationWireHelper {
```

Comment:
This looks extremely similar to configuration fetcher. Is there a significant difference? can this be just a new configuration fetcher?